Readability.js

```diff
@@ -867,6 +878,41 @@ Readability.prototype = {
           lastScore = parentOfTopCandidate.readability.contentScore;
           parentOfTopCandidate = parentOfTopCandidate.parentNode;
         }
+
+        // Find a better top candidate node if it contains (at least three) nodes which belong to `topCandidates` array
+        // and whose scores are quite closed with current `topCandidate` node.
+        var topCandidateAncestors = [];
+        topCandidates.some(function(_topCandidate, i) {
```

Comment:
In fact, it seems simpler to implement this as a raw for loop instead of using `some` with a function callback:

```js
for (let i = 1; i < topCandidates.length; i++) {
  if (/* score too low */) {
   break;
  }
  // add ancestors
}
```

using `some` (which is meant to answer a yes/no question about the array) in order to stop the iteration isn't the most obvious way of doing this.